client/src/components/DataFiles/DataFilesModals/DataFilesProjectEditDescriptionModal.js

```diff
@@ -0,0 +1,112 @@
+import React, { useCallback, useMemo } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import * as Yup from 'yup';
+import { Formik, Form } from 'formik';
+import FormField from '_common/Form/FormField';
+import { LoadingSpinner } from '_common';
+import { Button, Modal, ModalHeader, ModalBody } from 'reactstrap';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faExclamationCircle } from '@fortawesome/free-solid-svg-icons';
+import './DataFilesProjectEditDescription.module.scss';
+
+const DataFilesProjectEditDescriptionModal = () => {
+  const dispatch = useDispatch();
+  const isOpen = useSelector(state => state.files.modals.editproject);
+  const { title, description, projectId } = useSelector(
+    state => state.projects.metadata
+  );
+  const isUpdating = useSelector(state => {
+    return (
+      state.projects.operation &&
+      state.projects.operation.name === 'titleDescription' &&
+      state.projects.operation.loading
+    );
+  });
+  const updatingError = useSelector(state => {
+    return (
+      state.projects.operation &&
+      state.projects.operation.name === 'titleDescription' &&
+      state.projects.operation.error
+    );
+  });
+
+  const initialValues = useMemo(
+    () => ({
+      title,
+      description
+    }),
+    [title, description]
+  );
+
+  const toggle = () => {
+    dispatch({
+      type: 'DATA_FILES_TOGGLE_MODAL',
+      payload: { operation: 'editproject', props: {} }
+    });
+  };
+
+  const setProjectTitleDescription = useCallback(
+    values => {
+      dispatch({
+        type: 'PROJECTS_SET_TITLE_DESCRIPTION',
+        payload: {
+          projectId,
+          data: {
+            title: values.title,
+            description: values.description
+          }
+        }
+      });
+    },
+    [projectId, dispatch]
+  );
+
+  const validationSchema = Yup.object().shape({
+    title: Yup.string()
+      .min(1)
+      .required('Please enter a title.'),
+    description: Yup.string()
+  });
+
+  return (
+    <Modal size="lg" isOpen={isOpen} toggle={toggle} className="dataFilesModal">
+      <ModalHeader toggle={toggle}>Edit Descriptions</ModalHeader>
+      <ModalBody>
+        <Formik
+          initialValues={initialValues}
+          onSubmit={setProjectTitleDescription}
+          validationSchema={validationSchema}
+        >
+          <Form>
+            <FormField name="title" label="Workspace Title" />
+            <FormField
+              name="description"
+              label="Workspace Description"
+              type="textarea"
+              styleName="description-textarea"
+            />
+            <div styleName="button-container">
```

Comment:
@jchuahtacc , in comparison to the manage modal, I just stuck the button here without using the ModalFooter.  So it looks slightly different as there as the styling of ModalFooter has some things different than design (i.e. the horizontal line).  Should we remove ModalFooter from the create and manage modals?

Reply:
There's no footer on Manage, but I left it on Create because we have a create button.

Reply:
> I left it on Create because we have a create button.

![Screen Shot 2020-12-15 at 12 05 20 PM](https://user-images.githubusercontent.com/8287580/102254175-ef5d0280-3ecd-11eb-9263-949acbcc46a6.png)
I'm wondering about the horizontal line.

do you want me to change it here or someone else does it in separate pr or just leave as is?

Reply:
I'm not sure, because i just received a ui design regarding footers for the Make Link modal.

@tacc-wbomar ?

Reply:
Please, use the `ModalFooter`.

The `ModalFooter` is expected for any (exceptions unlikely) modal that has an action button at the bottom right.

_Craig has recently admitted to a habit of neglecting the modal footer line in his designs, even thought they are expected. I will let him know that neglecting them causing confusion for all developers, not just me._

Reply:
_**Update**: I have found **two** unresolved exceptions, two scrollable modals: manage account edit info, ticket create modal. Both these modals have the button in the **scrollable** modal body, and are (undesirably) flush against the bottom of the modal. The desired behavior can be defined by Design, and resovled in [FP-344: Create _common React Modal component](https://jira.tacc.utexas.edu/browse/FP-344)._

Reply:
👍 
The ticket-create is my mistake.  I think I got mixed up on the mockups that we didn't want ModalFooter.  We'll fix Edit Project Description modal before it the feature branch gets to master. Its on our list for that PR.